app/models/vocab/fedora_state.rb

```diff
@@ -0,0 +1,6 @@
+require 'rdf'
+module Vocab
+  class FedoraState < RDF::StrictVocabulary('http://fedora.info/definitions/1/0/access/')
```

Comment:
I copied this class in from https://github.com/pulibrary/plum/blob/master/app/models/vocab/f3_access.rb#L2, so @escowles be aware that you may have the same issue.